cypress/integration/login_spec.js

```diff
@@ -0,0 +1,74 @@
+/* 
+Arrange - setup inital app state
+    - vist a web page
+    - query for an element 
+Act - take action 
+    - interact with that element
+Assert - make an assertion 
+    - Make assertion on web page
+  */
+describe("Login", () => {
```

Comment:
work with @mhughes12 to write out the user stories related to authentication and name this according to the user story ID + AC ID name. Keep in mind, this should be the very first test that runs in our end-to-end test so name this spec so that it would be the first one.

Reply:
@CalvinFronda  I don't see this changed to A:00Login_spec in the later commits? You only changed the file name